include/wx/desktopenv.h

```diff
@@ -0,0 +1,33 @@
+//////////////////////////////////////////////////////////////////////////////
+// Name:        wx/desktopenv.h
+// Purpose:     wxDesktopEnv class interface
+// Author:      Igor Korot
+// Created:     2021-09-05
+// Copyright:   (c) 2021 wxWidgets development team
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_DESKTOPENV_H_BASE_
+#define _WX_DESKTOPENV_H_BASE_
+
+#include "wx/defs.h"
+
+class WXDLLIMPEXP_CORE wxDesktopEnvBase
```

Comment:
What's the point of having a "Base" class when there are no virtual functions here? This should be just `wxDesktopEnv` itself, it's a namespace-like class, it doesn't make any sense to inherit from it.